speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
Just making absolutely sure, since I definitely don't see the problem you're having with articles here..

You didn't happen to add the utterance, "play {Movie}," to the WatchMovie intent on a local branch or something, did you?  If you had, I could see that particular issue with ghostbusters that you noted.

If you do have any local changes you're testing on top of what's in the repo here, you might try removing them first.

I'm pressing this because I seriously use the WatchMovie intent on a near-daily basis and I *always* use articles in my speech when speaking to Alexa, because I find I stumble more if I try to omit them.

In fact, I actually had it on my own TODO list to "balance" the intents in this way, but the opposite of what you did.. I had planned to remove all articles from all utterances :P